app/assets/stylesheets/application.css.scss

```diff
@@ -35,4 +36,13 @@
 
 .block_user_current_block_id {
   margin-left: 20px;
-}
\ No newline at end of file
+}
+
+.flickr_search {
+  margin-bottom: 15px;
+}
+
```

Comment:
Avoid using id selectors<br>Selector `flickr_button` should be written in lowercase with hyphens